### ISSUE TITLE
home screen icons move up and down

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/equality-explorer#143]
Related to RC testing https://github.com/phetsims/QA/issues/144 and reported by @phet-steele.

Changes to joist to support more screens on the home screen apparently created a new problem. As you change the screen selection on the home screen, the unselected icons move up and down. See https://github.com/phetsims/joist/issues/470#issuecomment-405336702.  

@amanda-phet @ariel-phet Do you want this problem fixed for 1.0 release, or can we live with it?

**Comment by ariel-phet** [comment]
@pixelzoom we can live with this, since it is not happening in the regular sim. I understand it may affect translations, but this seems like a pretty minor aesthetic annoyance for the moment, and not pedagogically damaging or such (most people just go to the first screen anyhow from the homescreen from all our user testing experience). So it is not necessary to be fixed for the 1.0 release.

**Comment by amanda-phet** [comment]
I agree with everything @ariel-phet said. 

It's not a problem in *this* sim, but could be in a future 5-screen sim, so it should be handled at that time.

**Comment by pixelzoom** [comment]
@amanda-phet This has nothing to do with the number of screens.  It's a problem in all sims, noticeable in any sim with >= 3 screens.

**Comment by amanda-phet** [comment]
>It's a problem in all sims, noticeable in any sim with >= 3 screens.

Well it seems like something that should be handled, but perhaps not right now for this sim. It's @ariel-phet 's call on when.

**Comment by pixelzoom** [comment]
Agreed.  I just wanted to clarify that it's not caused by (or specific to) having 5 screens.

**Comment by pixelzoom** [comment]
Deferred, will not be addressed for 1.0 release.

**Comment by pixelzoom** [comment]
To be addressed in 1.0.1 maintenance release.

**Comment by jonathanolson** [comment]
Should I make the patch in the 1.0 branch for the sim?

**Comment by pixelzoom** [comment]
Is there another way to make a 1.0.1 maintenance release than in the 1.0 branch?

**Comment by jonathanolson** [comment]
> Is there another way to make a 1.0.1 maintenance release than in the 1.0 branch?

No, I just didn't know if you would prefer to make the change. I'm happy to.

**Comment by pixelzoom** [comment]
I won't be able to make the change until > August 5, and I believe that @ariel-phet would like it done sooner.   And there may be other sims that need to be patched.

**Comment by jonathanolson** [comment]
I'll take care of it, and I'll look up the commits to see about other sims that may also need the patch.

**Comment by pixelzoom** [comment]
@ariel-phet said something about masses-and-springs in https://github.com/phetsims/joist/issues/470#issuecomment-406009180.

**Comment by pixelzoom** [comment]
... and thanks for handling the patch.

**Comment by jonathanolson** [comment]
Equality explorer has been patched as part of https://github.com/phetsims/special-ops/issues/112 (waiting on testing). masses-and-springs was also patched.

Can we close this issue now, or leave open until the production deploys happen?

**Comment by pixelzoom** [comment]
@jonathanolson What is the status of this issue? https://github.com/phetsims/special-ops/issues/112 is now closed. Was this fixed in the 1.0.1 maintenance release?

**Comment by pixelzoom** [comment]
According to @jonathanolson via Slack... This issue was addressed in 1.0.1.  Closing.
